modules/mobile/app/controllers/mobile/v0/payment_history_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_dependency 'mobile/application_controller'
+require 'adapters/payment_shared_adapter'
+
+module Mobile
+  module V0
+    class PaymentHistoryController < ApplicationController
+      def index
+        start_date = params[:startDate] || (DateTime.now.utc.beginning_of_day - 1.year).iso8601
+        end_date = params[:endDate] || DateTime.now.utc.beginning_of_day.iso8601
```

Comment:
Should we use end of day here instead of beginning? Idk how the payments system works (maybe the timestamp doesn't matter), but it seems possible there could be payments that were made later in the day. If I had to guess, I'd speculate the start and end dates coming in from the params are probably dates and GetPaginatedList is coercing them into datetimes. In which case, maybe those get coerced to beginning of day too.

Reply:
End of day would probable make more considering the later payment within the same day. I will look into the payment systems more to determine why beginning of day way used for va.gov. The date params is used for the pagination (which is still WIP and will be finishing it this sprint).